kolibri/plugins/learn/assets/test/views/content-card-group-carousel.spec.js

```diff
@@ -125,13 +125,16 @@ describe('ContentCardGroupCarousel', function() {
           this.wrapper.setProps({contents});
           this.wrapper.instance().elSize.width = this.wrapper.instance().contentCardWidth * 3;
           // NOTE tried avoiding a nextTick in every test, didn't work. Dunno why.
```

Comment:
@rtibbles turns out it might have been working, since tests were failing? Or  maybe they would have failed regardless??